### PR TITLE
Vanilla Brewing Expanded

### PR DESCRIPTION
**Pull request by onigamii**
## Changes

Rewrote the alcohol patches to match the new structure of Brewing Expanded.

For testing purposes, here's my local version of [Vanilla Brewing Expanded](https://github.com/CombatExtended-Continued/CombatExtended/files/5087406/VanillaBrewingExpanded.zip)

**Currently Oskar is on holidays and the mod won't be updated until he comes back**

## Reasoning

Cider is added through a patch so we can't normally patch it like other drinks unless CE is loaded after Plants / Brewing Expanded, the case is that this isn't very user friendly and knowing that users will not always load CE after these mods and will cause errors we decided to add a new base drug that inherits from the vanilla drug that includes tools. If Brewing Expanded detects that Plants Expanded is installed and adds the cider to the game this won't error because the necessary patches will already be applied through the base drug def.

## TL;DR

- A new drug that inherits from the vanilla DrugBase will be added with tools included.
- All drinks with tools will change to this new base.
- CE will patch the new base tools.
- If Cider gets injected into the game, there won't be any tool related errors because it would be inheriting from the already patched no matter load order.

## Testing

- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested

### CODE REVIEW

**Comment by onigamii** [comment]
Here's the version of VBE that i modified  [VanillaBrewingExpanded.zip](https://github.com/CombatExtended-Continued/CombatExtended/files/5086067/VanillaBrewingExpanded.zip)

**Comment by N7Huntsman** [comment]
Out of curiosity, why is the Soda separated?

**Comment by onigamii** [comment]
> 
> 
> Out of curiosity, why is the Soda separated?

My bad, in my mind it was a soda can not a bottle.

**Review by Tostov — COMMENTED** [review]
Looks good on my end, didn't run into any immediate errors and the new tools work fine.
